### PR TITLE
Add Aikatsu! 365 Idol Days

### PR DESCRIPTION
**Pull request by PopuriAO29**
Adds Aikatsu! 365 Idol Days.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/aikatsu-365-idol-days/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Aikatsu! 365 Idol Days"
+description = "Aikatsu! 365 Idol Days is the third Aikatsu! game. It was developed by h.a.n.d. and published by Bandai Namco Games for the 3DS in 2014."
```
Comment (Leo626) [inline games/aikatsu-365-idol-days/game.dat]: `for the 3DS in 2014` is redundant in this description.

Comment (Leo626) [inline games/aikatsu-365-idol-days/game.dat]: Perhaps you can add more to the description like:

`The theme of the game is \"Real Idol Adventure.\" The game also includes characters from the season of the show that started airing earlier that Fall.`

**Comment by PopuriAO29** [comment]
where i put title screen?

**Comment by Leo626** [comment]
@PopuriAO29 I'm saying to rename the screenshots to be more descriptive. Not really add the title screen. So instead of `1.png` it could be named `menu.png` or something.

You make the changes to your own [forked repository](https://github.com/PopuriAO29/citra-games-wiki/tree/pr-builder-aikatsu-365-idol-days-72/games/aikatsu-365-idol-days) so that this PR will be updated.

**Comment by Leo626** [comment]
@PopuriAO29 One last thing to do is to address the comments on the description.

**Comment by PopuriAO29** [comment]
on games.dat file?

**Comment by Leo626** [comment]
Yes.

**Comment by PopuriAO29** [comment]
Alright, i'll see what happens goes.

**Comment by Leo626** [comment]
Just waiting for one more approval before it's merged.

**Pull request merged**